### ISSUE TITLE
Toaster top-center alignment doesn't have top margin

### ISSUE THREAD

**Issue by lscorcia** [scttcper/ngx-toastr#579]
Easily reproducible on https://scttcper.github.io/ngx-toastr/ , quickly trigger a default (top-right) toast, then a top-center one.

It seems a bit off that while the default top-right toaster alignment has margins on both left and top sides, the top-center does not have any top margin.

Even more puzzling, the bottom-center alignment option does not have the same problem and uses a small margin from the lower side of the viewport, as expected.

Is this a bug or is it by design? If the latter, can I override this behavior (I'm using the SCSS styles).
Thanks in advance

**Comment by scttcper** [comment]
i don't believe the styles intend for someone to use more than one position at a time

**Comment by lscorcia** [comment]
I'm sorry, my explanation was not clear. I didn't mean that I am applying more than one position at a time, I'm saying that the styles are not coherent one with the other. 

**Example: Top center alignment - No top-margin**
![image](https://user-images.githubusercontent.com/16761934/47914708-fc960f00-de76-11e8-8f2b-e98666aedb79.png)

**For reference: Top right alignment - A small top/right-margin from the viewport**
![image](https://user-images.githubusercontent.com/16761934/47914766-22231880-de77-11e8-9aaa-f2cc31dd43a6.png)

Thanks for your help

**Comment by oluwagbenga-joloko** [comment]
You can simply change the position of the toaster by adding this to the `styles.scss` file in the`src` folder of your application
``` CSS
.toast {
    top: 10px;
}
```
The `toast` class is applied to the toaster element. You just need to adjust its positioning.

**Comment by lscorcia** [comment]
I checked the source of the styles at `src/lib/toastr-bs4-alert.scss` and `src/lib/toastr.css` and it actually seems like it's intentional, even if a bit awkward on screen. Only the corner toasters have the 12px spacing from the viewport border.

Anyway thanks to the comment by @oluwagbenga-joloko I applied the following to my `src/styles.scss` and obtained the desired behaviour:

```css
.toast-top-center {
  top: 12px;
}
```

I'm closing the issue since it really seems that the behaviour is by-design.
Thanks for your help!
